src/schema/broadband-signup-us.json

```diff
@@ -281,6 +285,107 @@
                             "default": ""
                         }
                     },
+                    "document": {
+                        "type": "object",
+                        "description": "Person ID (passport or other document). Automation may fail if this information is required by the website, but not provided.",
+                        "pii": true,
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "description": "Document type.",
+                                "enum": [
```

Comment:
Are all of these supported by the supplier?

Reply:
Yes.